### PR TITLE
Fixing libraries for PT-BR

### PR DESCRIPTION
**Pull request by RCastilho91**
Full review for lib/PT-BR made. Log change and reasoning are as follows:

# lib/locales/pt_BR/address/country.js

## Changed
"Anguilla" -> Anguila
"Antigua and Barbada" -> Antígua e Barbuda
"Alzerbaijão" -> Azerbaijão
"Barbado" -> Barbados
"Bhutan" -> Butão
"Bôsnia" -> Bósnia
"Botuasuna" -> Botsuana
"Bouvetoia" -> Ilha Bouvet
"Cambójia" -> Camboja
"Chad" -> Chade
"Ilhas Natal" -> Ilha do Natal
"Comoros" -> Comores
"Cyprus" -> chipre
"Djibouti" -> Jibuti
"Eritrea" -> Eritreia
"Guernsey" -> Guernesey
"Heard Island and McDonald Islands" -> Ilhas Heard e McDonald
"Iceland" -> Islândia
"Kiribati" -> Quiribati
"Kyrgyz Republic" -> Quirguistão
"República Democrática de Lao People" -> Laos
"Latvia" -> Letônia
"Lesotho" -> Lesoto
"Libyan Arab Jamahiriya" -> Líbia
"Mauritius" -> Maurícia
"Mayotte" -> Maiote
"Moldova" -> Moldávia
"Northern Mariana Islands" -> Marianas Setentrionais
"Oman" -> Omã
"Nova Guiné Papua" -> Papua-Nova Guiné
"Puerto Rico" -> Porto Rico
"Saint Martin" -> São Martinho
"Saint Pierre and Miquelon" -> São Pedro e Miquelão
"Saint Vincent and the Grenadines" -> São Vicente e Granadinas
"Seychelles" -> Seichele"
"South Georgia and the South Sandwich Islands" -> Ilhas Geórgia do Sul e Sandwich do Sul
"Spanha" -> Espanha
"Svalbard & Jan Mayen Islands" -> "Ilhas Svalbard & Jan Mayen"
"Swaziland" -> Suazilândia
"Tokelau" -> Toquelau
"Trinidá e Tobago" -> Trinidad e Tobago
"Turks and Caicos Islands" -> Turcas e Caicos
"Wallis and Futuna" -> Wallis e Futuna
"Yemen" -> Iêmen

## Removed
"Belgrado" - Is actually Servia's capital, not a country
"Guano" - No such country. Guano is the accumulated excrement of seabirds and bats
"Sahara" - The Sahara desert spans several North African countries and can be considered no country on its own

# lib/locales/pt_BR/address/state_abbr.js

## Added
- MG (state of Minas Gerais)
- SE (state of Sergipe)
- TO (state of Tocantins)

# lib/locales/pt_BR/company/suffix.js

## Added
- EIRELI

# lib/locales/pt_BR/address/name/prefix.js

## Added
- Dra.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#232E41",
+        "titleBar.activeBackground": "#31405B",
+        "titleBar.activeForeground": "#F9FAFC"
+    }
+}
```
Comment (pgcalixto) [inline .vscode/settings.json:7]: I believe this should not be committed.

File: lib/locales/pt_BR/address/country.js [hunk 2]
```diff
@@ -158,51 +156,51 @@ module["exports"] = [
...
-  "South Georgia and the South Sandwich Islands",
-  "Spanha",
+  "Ilhas Geórgia do Sul e Sandwich do Sul",
+  "Espanha",
   "Sri Lanka",
   "Sudão",
   "Suriname",
-  "Svalbard & Jan Mayen Islands",
-  "Swaziland",
+  "Ilhas Svalbard & Jan Mayen",
```
Comment (pgcalixto) [inline lib/locales/pt_BR/address/country.js:202]: Shouldn't it be "Ilhas Svalbard **e** Jan Mayen"?

**Review by RCastilho91 — COMMENTED** [review]
Got rid of a configuration file that was accidentally committed

**Comment by pgcalixto** [comment]
Kudos for updating pt-br locales! :D

File: lib/locales/pt_BR/address/country.js (hunk 2 above)
Comment (RCastilho91) [reply lib/locales/pt_BR/address/country.js:202]: I tried keeping the code as untouched as possible, removing only actual problems. Originally "&" could be replaced, but it stands for the same purpose.

**Comment by pgcalixto** [comment]
Isn't this a duplicate of https://github.com/Marak/faker.js/pull/997?

**Comment by RCastilho91** [comment]
#997 reviews only country names and less thoroughly than my review. I have reviewed the entire folder for any issues. Should the maintainers deem the country name effort unnecessary, they may still accept the remaining files.

File: lib/locales/pt_BR/address/country.js (hunk 2 above)
Comment (pgcalixto) [reply lib/locales/pt_BR/address/country.js:202]: Makes sense :)

**Review by JJnrw79 — APPROVED** [review]

**Review by JJnrw79 — APPROVED** [review]

**Review by JJnrw79 — COMMENTED** [review]
logo

**Comment by Marak** [comment]
Thank you for your contributions @RCastilho91 !

Merged.

**Pull request merged**